sitemap.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <url>
+    <loc>https://okturtles.org/</loc>
+    <priority>0.8</priority>
+  </url>
+
+  <url>
+    <loc>https://okturtles.org/dpki</loc>
+  </url>
+
+  <url>
+    <loc>https://okturtles.org/donate</loc>
+    <priority>0.7</priority>
+  </url>
+
+  <url>
+    <loc>https://okturtles.org/team</loc>
+  </url>
+
+  <url>
+    <loc>https://okturtles.org/finances</loc>
+  </url>
```

Comment:
Can you remove the finances page from here?

Reply:
@taoeffect sure, if you would like to make it unnoticeable by search engine crawlers, I can also add `noindex` meta tag for `/finances` page too, just like we did in the previous PR.

Reply:
I don't think that's necessary, unless we discover that it's pushing out one of the 4 pages we want displayed in the search summary results